packages/rum/package.json

```diff
@@ -12,7 +12,7 @@
     "build:esm": "rm -rf esm && tsc -p tsconfig.esm.json"
   },
   "dependencies": {
-    "@datadog/browser-core": "4.45.0",
+    "@datadog/browser-core": "https://gitpkg.now.sh/corona-school/browser-sdk/packages/core?lern-fair-features",
```

Comment:
This will make sure that the rum package will use our updated browser-core package.